fabric/state.py

```diff
@@ -224,6 +224,13 @@ def _rc_path():
         help="default to parallel execution method"
     ),
 
+    make_option('--parallel-exit-on-errors',
```

Comment:
It would be nice to avec a short option version. `-E` seems to be free.

What do you think?